js/src/lib/Details/Cdn.js

```diff
@@ -47,16 +47,34 @@ const BundleRun = ({ name, version }) => (
 );
 
 class Cdn extends Component {
+  constructor(props) {
+    super(props);
+
+    this.cdns = {
+      jsdelivr: JsDelivr,
+      unpkg: Unpkg,
+      bundlerun: BundleRun,
+    };
+
+    this.order = Object.keys(this.cdns).sort(() => {
+      return Math.random() - 0.5;
+    });
+  }
+
   render() {
+    const items = this.order.map(key =>
+      React.createElement(this.cdns[key], {
```

Comment:
You could also assign it to a variable with a capital name and use it as a component.

Reply:
^ Yeah this is my preference, too

```
const items = this.order.map(key => {
 const CDNComponent = this.cdns[key];
  return (
    <CDNComponent
      key={key}
      ........
    />
  );
}
```

Reply:
JSX compiles down to `createElement`. Don't be afraid to use the JavaScript API when you've got JavaScript variables, guys 😅

Reply:
@mjackson - Consistency is good, though. Why use `React.createElement` at just one call site?